src/circleci.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Provides methods for fetching data from CircleCI's API.
+ * @flow
+ */
+
+import got from 'got';
+import url from 'url';
+
+export type Build = {
+  pull_requests?: [{
+    head_sha: string,
+    url: string,
+  }],
+};
+
+const headers = {
+  Accept: 'application/json',
+  'User-Agent': 'BabelBot/1.0.0'
+};
+const API_BASE = 'https://circleci.com/api/v1.1/project/github';
```

Comment:
For Yarn, our release infra doesn't use an access token for CircleCI either, and it seems to be working fine: https://github.com/yarnpkg/release-infra/blob/master/lib/CircleCI.php#L100-L110